baseplate/context/memcache/lib.py

```diff
@@ -0,0 +1,111 @@
+import logging
+import zlib
+
+from ..._compat import (
+    long,
+    pickle,
+    string_types,
+)
+
+
+"""Memcache serialization/deserialization (and compression) helper methods.
+
+Memcached can only store strings, so to store arbitrary objects we need to
+serialize them to strings and be able to deserialize them back to their
+original form. General purpose serialization and deserialization can be
+achieved with pickle_and_compress() and decompress_and_unpickle().
+
+"""
+
+
+FLAG_PICKLE = 1 << 0
```

Comment:
I think a comment got lost in the shuffle here, can this be done with the py3.4 style Enum stuff? (there's a shim in _compat for it).